base_layer/core/src/transactions/transaction_components/transaction_kernel.rs

```diff
@@ -100,13 +105,19 @@ impl TransactionKernel {
             lock_height,
             excess,
             excess_sig,
+            burn_commitment,
         )
     }
 
     pub fn is_coinbase(&self) -> bool {
         self.features.contains(KernelFeatures::COINBASE_KERNEL)
     }
 
+    /// Is this a burned output kernel?
+    pub fn is_burned(&self) -> bool {
+        self.features.contains(KernelFeatures::BURNED_KERNEL)
+    }
```

Comment:
nit: To me, it's strange to mark/flag in the past tense (this is a burn kernel) and `BURN` is consistent with other flag names (e.g. you wouldn't say: coinbased)
```suggestion
    /// Returns true if the output kernel is marked as `BURN`, otherwise false
    pub fn is_burn(&self) -> bool {
        self.features.contains(KernelFeatures::BURN_KERNEL)
    }
```

Happy if you want to keep it as BURNED though